src/components/MessageActionsList/MessageActionsList.types.ts

```diff
@@ -0,0 +1,30 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import { BaseIcon } from '../Icon';
+
+export interface MessageActionsListProps extends BaseComponentProps {
+  /**
+   * Items to show in the list.
+   */
+  items: MessageActionsListItem[];
+
+  /**
+   * Aria label for the overflow menu button.
+   */
+  overflowMenuAriaLabel: string;
+
+  /**
+   * The maximum number of items to display in the list.  The remaining will show in the menuButton.
+   */
+  maxVisibleItemCount?: number;
+}
+
+export type MessageActionsListItem = {
```

Comment:
seems like we're duplicating ActionLink props, but losing the nuance of it's compound prop types (link or clickable). should we remove this interface and set line 9 to `items: ActionItemProps[]`?

Reply:
Yeah, I agree this could be better. There are two things that lead me to create the new interface here. 
1. ActionLink has an option `ariaLabel`, `menuButton` requires it.
2. Currently unlinkedText is rendered by the list component, not the ActionLink. 

I'll try to figure something out.

Reply:
so menuButton options have to be onclick handlers, not hyperlinks?

Reply:
your call on this, whether it makes sense to extend ActionItemProps[] or not